test/utils/createIdentityProof.testSuite.ts

```diff
@@ -0,0 +1,29 @@
+import { IAM } from "../../src/iam";
```

Comment:
claimsTests seems like a more proper place for this tests

Reply:
I Agree